javascript/event-recommender.md

```diff
@@ -0,0 +1,154 @@
+# Event Recommender System with Javascript
+
+
+### Primary Goals
+To build a modular system using Object Oriented Programming principles.
+
+### Overview
+In this project, you'll build an EventRecommender System that will
+help you practice Object Oriented Programming using Javascript. This should take
+a maximum of 3 hours.
+
+### Context
+* Object Oriented Programming (OOP) is a popular style of programming that is oftentimes
+adopted for Backend Code Organization.
+* OOP enables
+    * Readability (esp in a large code base)
+    * Function abstraction
+    * Easier testing
+* Displaying Object Oriented Programming Skills in an interview is extremely impressive, and shows your ability to design and organize systems.
+
+
+### How to get Started
+#### User Requirements
+When you instantiate an Event recommender object, you should be able to do the following:
+
+* Users:
+    * Add a new User to the System
+    * Search for the User
+    * Save the Events that User wish to go
+    * Delete a User from the System
+* Event:
+    * Add a new Event to the System
+    * Update an Eventt's categories
+    * Add comment to the Event
+    * Maintain a Event's "average" score every time a new user score is added.
+    * Delete an Event from the System
+* Category:
+    * Add a new Category to the System
+    * Update Events in a Category
+    * Delete a Category from the System
+* Venue:
+    * Add Venue to the System
+    * Modify Venue details
+    * Search for Venue as per choice
+    * Delete Venue from the System
+* Performer:
```

Comment:
Can you tell me where this performer stuff comes from? I think it may be unnecessary.

Reply:
Please do not add methods, we are trying to make this project easier! Students will have around 4 weeks of coding experience. The original project had 8 methods - please don’t have any more than that.

We are trying to change the restaurant  project and possibly simplify it. If you added anything extra to these files please remove it.